zuchini-runner/src/main/java/org/zuchini/runner/Converters.java

```diff
@@ -11,46 +11,82 @@
 
 import static java.util.Arrays.asList;
 
-public class Converters {
+class Converters {
 
-    static enum DefaultConverters implements Converter {
-        BYTE() {
+    enum DefaultConverters implements Converter<Object> {
+        BYTE_PRIMITIVE {
             @Override
             public Object convert(String argument) {
```

Comment:
The individual implementations could return their respective subtype, e.g. `Byte` in this case.

Reply:
It's not type safe either way, unfortunately enum constants don't support specifying generic parameters.

Reply:
Wouldn't it help in case someone calls e.g. `DefaultConverters.BOOLEAN.convert("true")` directly?